site/docs/spec.md

```diff
@@ -267,7 +267,8 @@ A sort order is defined by an sort order id and a list of sort fields. The order
 
 Order id `0` is reserved for the unsorted order. 
 
-Sorting floating-point numbers should produce the following behavior: `-NaN` < `-Infinity` < `-value` < `-0` < `0` < `value` < `Infinity` < `NaN`. This aligns with the implementation of Java floating-point types comparisons. 
+Sorting floating-point numbers should produce the following behavior: `-Infinity` < `-value` < `-0` < `0` < `value` < `Infinity` < `NaN`.
+The different `NaN` representation can be in an arbitrary order. This aligns with the implementation of Java floating-point types comparisons.
```

Comment:
can you specify more on what "order" means here?

Reply:
"order" was supposed to refer to sorting from the previous sentence. can i ask you for help on better wording?